src/App.js

```diff
@@ -1,29 +1,69 @@
 import './App.css';
+import '../node_modules/bootstrap-icons/font/bootstrap-icons.css'
+import {useState} from 'react'
+import {render} from 'react-dom'
+let list=[];
+const input=document.getElementById('input');
+
+let btnFunction='new';
+let clickAction={
+  'edit':function(e){
+    e.preventDefault();
+    console.log(e.target.attributes.index.value)
+  },
+'delete':function(e){
+  e.preventDefault();
+  console.log(e.target.attributes.index.value)
+
+}
+}
+
+function Itemslist (props){
+  let itemsarray=props.list
+  if(itemsarray.length>0){
+  const items=itemsarray.map((item,index)=>(<li className="list-group-item d-flex justify-content-between align-items-center" >
```

Comment:
This could be an expensive computation, imagine you have a 1000 items here, is it reasonable to recompute the value on EVERY render?
Use React's `useMemo` to compute expensive values only when needed.